checkpoint_quantum_firewall/assets/logs/checkpoint-quantum-firewall.yaml

```diff
@@ -0,0 +1,946 @@
+    path: checkpoint.quantum.firewall.client_outbound_packets
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Confidence Level
+    path: checkpoint.quantum.firewall.confidence_level
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Connection Direction
+    path: checkpoint.quantum.firewall.conn_direction
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Destination
+    path: checkpoint.quantum.firewall.dst
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Event Type
+    path: checkpoint.quantum.firewall.event_type
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Host Type
+    path: checkpoint.quantum.firewall.host_type
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Identity Source
+    path: checkpoint.quantum.firewall.identity_src
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Identity Type
+    path: checkpoint.quantum.firewall.identity_type
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Layer Name
+    path: checkpoint.quantum.firewall.layer_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Malware Action
+    path: checkpoint.quantum.firewall.malware_action
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Malware Rule Name
+    path: checkpoint.quantum.firewall.malware_rule_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Matched Category
+    path: checkpoint.quantum.firewall.matched_category
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Operation
+    path: checkpoint.quantum.firewall.operation
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: OS Name
+    path: checkpoint.quantum.firewall.os_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Protection Name
+    path: checkpoint.quantum.firewall.protection_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Protection Type
+    path: checkpoint.quantum.firewall.protection_type
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Protocol
+    path: checkpoint.quantum.firewall.protocol
```

Comment:
Based on the sample it seems that this attribute could be remapped to a standard attribute. See: Can you check if this fits into some of the standard attributes please ? https://docs.datadoghq.com/standard-attributes/?search=protocol

Reply:
The checkpoint docs are not clear to mention if the protocol value is limited to http or https hence we have refrained from mapping the same.